Packs/AggregatedScripts/Scripts/GetEndpointData/GetEndpointData.py

```diff
@@ -86,35 +125,113 @@ def is_brand_available(self, command: Command) -> bool:
         return False if not self.is_brand_in_brands_to_run(command) else command.brand in self._enabled_brands
 
 
+def filter_empty_values(input_dict: dict) -> dict:
```

Comment:
why not using csp's remove_empty_elements?

Reply:
Because `remove_empty_elements` does not refer to an empty string.

Reply:
Then let's add support for that, I think it makes more sense than duplicating the function

Reply:
Changing the CSP `remove_empty_elements`  may break backward support for many integrations and scripts that use it.